tests/model_fields/tests.py

```diff
@@ -661,6 +689,15 @@ def test_backend_range_validation(self):
             }
             with self.assertRaisesMessage(ValidationError, expected_message):
                 instance.full_clean()
+            try:
+                with transaction.atomic():
+                    instance.save()
+            except (DataError, IntegrityError):
+                pass  # backends such as Postgres don't fail silently
+            else:
+                qs = self.model.objects.filter(value__lte=min_value)
```

Comment:
The Oracle backend is included in Django so we could certainly test that without much difficulty. There are a few different SQL server backends. https://bitbucket.org/Manfre/django-mssql is the main one I know about (it's author is on the Django team) -- it's still being updated for Django 1.8 though.